apps/bemo-worker/src/BemoDO.ts

```diff
@@ -35,4 +52,150 @@ export class BemoDO extends DurableObject<Environment> {
 			})
 		}
 	}
+
+	async handleConnect(req: IRequest) {
+		// extract query params from request, should include instanceId
+		const url = new URL(req.url)
+		const params = Object.fromEntries(url.searchParams.entries())
+		const { sessionKey } = params
```

Comment:
```suggestion
		const { sessionKey } = req.query
```

Or use `parseRequestQuery` which comes with error handling